includes/templates/template_default/templates/tpl_checkout_confirmation_default.php

```diff
@@ -175,7 +175,7 @@
         ?>
 
         <?php
-        echo zen_draw_form('checkout_confirmation', $formActionLink, 'post', 'id="checkout_confirmation" onsubmit="submitonce();"');
+        echo zen_draw_form('checkout_confirmation', $formActionLink, 'post', 'id="checkout_confirmation" onsubmit="submitonce();"' . (defined('MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE') && MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE == 'InContext' && defined('MODULE_PAYMENT_PAYPALWPP_MERCHANTID')    && MODULE_PAYMENT_PAYPALWPP_MERCHANTID    != '' && defined('MODULE_PAYMENT_PAYPALWPP_STATUS')        && MODULE_PAYMENT_PAYPALWPP_STATUS        == 'True' ? ' data-paypal-button="true" data-paypal-id="' . MODULE_PAYMENT_PAYPALWPP_MERCHANTID . '"':''));

```

Comment:
A separate function would/could provide the desired result, but seems would have to test for the presence of that function unless going to add it to all of the existing payment modules... Is a little "selfish" for it to just be for PayPal to be able to add to the innerhtml of the tags.